nerfstudio/cameras/lidars.py

```diff
@@ -35,6 +36,7 @@
 from nerfstudio.utils.misc import strtobool, torch_compile
 from nerfstudio.utils.tensor_dataclass import TensorDataclass
 
+#torch._dynamo.config.suppress_errors = True
```

Comment:
Please remove unused code :)

Reply:
Will removed this line along with the one above.

Reply:
Done